src/crypto/SymmetricCipher.cpp

```diff
@@ -104,6 +104,20 @@ SymmetricCipher::Algorithm SymmetricCipher::cipherToAlgorithm(const QUuid& ciphe
     return InvalidAlgorithm;
 }
 
+QString SymmetricCipher::cipherToString(const QUuid& cipher)
```

Comment:
Technically this information is available in the `KeePass2::CIPHERS` list which I recommend using to standardize the names.

Reply:
good call, I used that instead.